Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM --platform=linux/amd64 python:3.10.6-buster
+
```

Comment:
I get a strange warning here the same with my code, "The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested" maybe just switching it to --platform=linux/amd64/v8 could resolve future conflicts